src/lib/api.tsx

```diff
@@ -27,14 +28,17 @@ export const initialiseApi = ({
 	baseUrl,
 	additionalHeaders,
 	apiKey,
+	idApiUrl,
 }: {
 	baseUrl: string;
 	additionalHeaders: AdditionalHeadersType;
 	apiKey: string;
+	idApiUrl: string;
 }) => {
 	options.baseUrl = baseUrl || options.baseUrl;
 	options.headers = additionalHeaders || options.headers;
 	options.apiKey = apiKey || options.apiKey;
+	options.idApiUrl = idApiUrl || options.idApiUrl;
```

Comment:
is this fallback intended to guard against the case where the `idApiUrl` passed on line 31 is an empty string?

Reply:
Good question! Yes, I believe that would be the empty string. I am not entirely clear about the full flow, i.e. how this value ends up in CAPI, but I'm guessing an empty string could happen, so we're adding this default case. Also, this was we are following the existing pattern.